book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] We need both   _Getters_ and _Setters_ for _@rentals_ to be present, therefore kindly ensure to mark _rentals_ with _attr_accessor_, like:
```ruby
  attr_accessor :rentals

Reply:
@rloterh I have a question, why do I need a setter for `:rentals: if I need to create a method `add_rental`?
Won't I only need a getter?

Reply:
@guerreiropedr0 
Yes both _setters_ and _getters_ are **required** for :rentals
Cheers!

Reply:
@rloterh Can you tell me why? I still don't know and I would like to understand

Reply:
@guerreiropedr0
 
When you create an instance of a Book, it will have the `rentals` (which is an array) because it's an instance variable in the constructor and this makes it possible for the has_many side of Book and Rental (a book has many rentals). Remember that in the `Rental` class, you're pushing a rental to that rentals array that is present in the Book class, so you need getter and setter to be able to add rentals and also be able to display all rentals a book has. This can also be said to read and write rentals. I hope this helps.

Reply:
Thank you